openedx/adg/admin_dashboard/applications/tests_applications/test_applications_helpers.py

```diff
@@ -0,0 +1,135 @@
+"""
+Tests for all the helper functions in `admin_dashboard.applications` app.
+"""
+import json
+from datetime import datetime, timedelta
+
+import pytest
+
```

Comment:
Import order is correct?

Reply:
Yes it is correct, otherwise isort would have given an error. Plus I sorted imports all the files using isort

Reply:
I think we have added extra blank lines.

Reply:
The blank lines above and below `import pytest` were added my isort.